ledger/ledger-api-tests/infrastructure/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/SingleParticipantTestContext.scala

```diff
@@ -218,20 +226,35 @@ final class SingleParticipantTestContext private[participant] (
 
   override def allocateParty(
       partyIdHint: Option[String],
-      displayName: Option[String],
+      displayName: Option[String] = None,
```

Comment:
It's a bit strange to have a default value on a pen-ultimate position: either move to the end or provide default for the last arg.

Reply:
Adding default values for all three parameters